core/src/main/java/cz/o2/proxima/storage/StorageFilter.java

```diff
@@ -68,4 +75,22 @@ public boolean apply(StreamElement ingest) {
    * @return {@code false} to throw the element away
    */
   boolean apply(StreamElement ingest);
+
+  /**
+   * Setup {@link AttributeFamilyDescriptor} filter.
+   *
+   * @param af family descriptor
+   */
+  default void setup(AttributeFamilyDescriptor af) {
```

Comment:
It seems unclear how to use this method. Looking at how a `Transformation` uses this, could we do something similar? I.e.
```java
  void setup(Repository repo, Map<String, Object> cfg)
```

that way we could use single `setup` for both cases - transformation and replication. That would make more sense to me.

Reply:
Changed.